src/server/api/controllers/families.controller.js

```diff
@@ -0,0 +1,25 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getFamilies = async () => knex('families');
+
+
+const getFamilyById = async (id) => {
+  if (!id) {
+    throw new HttpError('Id should be a number', 400);
```

Comment:
I test this in swagger and for some reason it give back an error 200 instead of 400 ?!
![Screen Shot 2022-03-02 at 12 42 40 PM](https://user-images.githubusercontent.com/77116575/156356563-b40f4862-2d82-4265-966c-5a740b2a4685.png)